website/search/elastic_search.py

```diff
@@ -294,8 +296,13 @@ def update_node(node, index=None, bulk=False):
         else:
             es.index(index=index, doc_type=category, id=elastic_document_id, body=elastic_document, refresh=True)
 
+    #TODO @hmoco, make this a celry task that takes care of updating files
```

Comment:
Just for clarification: this function is run in celery task already, so the calls to `update_file` will not block the main app processes, correct @hmoco ?